src/vsl/parser/grammar/class.ne

```diff
@@ -38,11 +38,11 @@ ExtensionList
    -> delimited[type {% id %}, _ "," _] {% id %}
 
 ClassItems[s]
-   -> CodeBlock[ClassItem[$s] {% id %}] {% id %}
+   -> CodeBlock[ClassItem[$s {% id %}] {% id %}] {% id %}
 ClassItem[s]
-   -> FunctionStatement[$s] {% id %}
+   -> FunctionStatement[$s {% id %}] {% id %}
 
 InterfaceItems[s]
-   -> CodeBlock[(InterfaceItem | ClassItem[$s]) {% mid %}] {% id %}
+   -> CodeBlock[InterfaceItem {% id %}] {% id %}
```

Comment:
This PR is meant to fix the grammar for classes and interfaces, not necessarily to extend it. I'll add `FunctionStatement`s back to `InterfaceItem` but I don't think it's within the right context to start adding new features to the grammar just yet.